Fixes #153 - Code quality/optimization

RazCrimson opened the pull request:
Fixes #153 , I have made the corrections. Please review it @rgroves .

RazCrimson committed:
Removing redundant initialization

RazCrimson committed:
Fixing a typo for mongodb in index.js

RazCrimson committed:
Added awaits with async function calls

RazCrimson committed:
Remove redundant variable usage

rgroves commented on src/commands/poll.js:
```diff
@@ -68,10 +68,7 @@ module.exports = {
-            minutes = 0
-            embed.setFooter(`The poll has started and has no end time`)
-        }
+        } else embed.setFooter(`The poll has started and has no end time`)
```
Let's keep the else side's code block (`{...}`) even though there is only one statement being executed. This is done in other parts of the codebase, so let's keep it consistent. Update the following:
```
        } else embed.setFooter(`The poll has started and has no end time`)
```
...to this...
```
         } else {
             embed.setFooter(`The poll has started and has no end time`)
         }
```

rgroves replied:
Hey I just went ahead and added this since it was a small change and I had time.

rgroves reviewed (changes requested):
Just one small change and this will be accepted and merged.

rgroves committed:
Add braces to else block.

Just keeping things consistent with other single statement if/else blocks in the codebase.
